StringMethods.java

```diff
@@ -1 +1,39 @@
+package com.main;
 
+/**
+ Simple program to demonstrate the use of differents methods of string
+ Practical 10 from core java practicals 
+ @author Mashuk Patel
+ */
+public class StringMethods {
+
+	/**
+	 method to check whether both the string are same or not
+	 @param firstname first string
+	 @param lastname second string
+	 @return true if both the string are same otherwise false
+	 */
+	public static boolean isEqual(String firstname, String lastname) {
+		if(firstname.equals(lastname)) {
+			return true;
+		}
+		else {
+			return false;
+		}
+	}
+	
+	public static void main(String[] args) {
+		String firstname="Mashuk";
```

Comment:
1) Variable name must be in camel case.
2) Please apply the valid space after variable firstName = "Mashuk";
check this changes in all places